brave/brave-core
# Fix pinned tab container

* Make Only unpinned tab container scrollable
* Move views::ScrollView from VerticalTabStripRegionView to BraveCompoundTabContainer to split scrollable area.


https://github.com/brave/brave-core/assets/5474642/928c0f9c-f277-4949-846b-0b4d931d302d




Resolves https://github.com/brave/brave-browser/issues/29935



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Enable brave://flags/#brave-vertical-tabs-stick-pinned-tabs and vertical tab strip
* Add tabs until tabs are overflowed
* Check if pinned tabs are not scrolled when scrolling tabs.
